src/cli.js

```diff
@@ -10,38 +10,41 @@
  * governing permissions and limitations under the License.
  */
 
-const fs = require('fs');
-
+// declared dependencies
+const fse = require('fs-extra');
```

Comment:
just name it `fs`

Reply:
ok, i can do that. but aren't you a bit picky?

quoting [node-fs-extra](https://github.com/jprichardson/node-fs-extra#usage):

if you prefer to make it clear that you're using `fs-extra` and not `fs`, you may want to name your `fs` variable `fse` like so:
```javascript
const fse = require('fs-extra')
```